cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/AbstractOperator.java

```diff
@@ -572,38 +566,38 @@ private Future<Void> handleResult(Reconciliation reconciliation, AsyncResult<Voi
      */
     private Future<Void> updateResourceState(Reconciliation reconciliation, boolean ready, Throwable cause) {
         String key = reconciliation.namespace() + ":" + reconciliation.kind() + "/" + reconciliation.name();
+        String errorReason = cause == null
+                ? "none"
+                : cause.getMessage() == null
+                ? "unknown error"
+                : cause.getMessage();
```

Comment:
I do love ternary operator and I use it a lot but ... I think that a ternary operator within another one is difficult to ready. I would use a `if` condition instead. But it's just my persona opinion.

Reply:
Was already like that: https://github.com/strimzi/strimzi-kafka-operator/pull/9861/files/c48cf3937b725001840fb3cee2cc0ec410389b41#diff-67a56b74dbdb678b1883c7d5b3eaf7b1ad0b2f2213c7179da3b6fe05bc12b386L580 but I happily rewrite it to be more meaningful :)